### ISSUE TITLE
Is the normalizatin values for CIFAR-10 correct?

### ISSUE THREAD

**Issue by abduallahmohamed** [gpleiss/efficient_densenet_pytorch#68]
Hi, 
I believe that the CIFAR-10 normalization values used are actually for CIFAR-100. 

The current values are (which actually are CIFAR-100 normalization values): 
```python
mean = [0.5071, 0.4867, 0.4408]
stdv = [0.2675, 0.2565, 0.2761]
```
But they should be: 
```python
mean=[0.49139968  0.48215841  0.44653091]
stdv= [0.24703223  0.24348513  0.26158784]
```

Ref: https://github.com/Armour/pytorch-nn-practice/blob/master/utils/meanstd.py

**Comment by gpleiss** [comment]
That seems correct. Shouldn't make a difference in practice w/ batch norm.
I'd be happy to accept a PR fixing these numbers
